diff --git a/node_modules/@opentripplanner/humanize-distance/esm/index.js b/node_modules/@opentripplanner/humanize-distance/esm/index.js
index 1c5807e..b9b6154 100644
--- a/node_modules/@opentripplanner/humanize-distance/esm/index.js
+++ b/node_modules/@opentripplanner/humanize-distance/esm/index.js
@@ -40,9 +40,10 @@ export function humanizeDistanceStringMetric(meters, intl) {
     unitDisplay: "short"
   }) : "".concat(value, " ").concat(shortUnit);
 }
-export function humanizeDistanceString(meters) {
-  var outputMetricUnits = arguments.length > 1 && arguments[1] !== undefined ? arguments[1] : false;
-  var intl = arguments.length > 2 ? arguments[2] : undefined;
-  return outputMetricUnits ? humanizeDistanceStringMetric(meters, intl) : humanizeDistanceStringImperial(meters, null, intl);
+export function humanizeDistanceString(meters, outputMetricUnits, intl) {
+  return humanizeDistanceStringMetric(meters, intl);
+  // var outputMetricUnits = arguments.length > 1 && arguments[1] !== undefined ? arguments[1] : false;
+  // var intl = arguments.length > 2 ? arguments[2] : undefined;
+  // return outputMetricUnits ? humanizeDistanceStringMetric(meters, intl) : humanizeDistanceStringImperial(meters, null, intl);
 }
 //# sourceMappingURL=index.js.map
